src/App.js

```diff
@@ -39,7 +42,9 @@ function App() {
               <LoginForm />
             </Route>
             <Route path="/users/:id">
-              <UserContainerRedirector />
+              <CookieOrRedirect redirectPath="/users/login">
```

Comment:
May I ask why you chose to prefix with Cookie again? A user could have multiple cookies on their browser

Reply:
Perhaps AuthRedirect ?

Reply:
Thats true, I used that one mainly because it was descriptive but AuthRedirect would probably be more accurate to it's function